gems/aws-sdk-core/spec/aws/ecs_credentials_spec.rb

```diff
@@ -165,33 +118,79 @@ module Aws
           end.to raise_error(ArgumentError, /without a credential path/)
         end
       end
-    end
 
-    context 'retries' do
-      before(:each) do
-        ENV['AWS_CONTAINER_CREDENTIALS_RELATIVE_URI'] = path
-      end
+      context 'retries' do
+        it 'defaults to 0' do
+          stub_request(:get, "http://169.254.170.2#{path}").to_raise(SocketError)
+          expect(ECSCredentials.new(backoff: 0).retries).to be(5)
+        end
 
-      it 'defaults to 0' do
-        stub_request(:get, "http://169.254.170.2#{path}").to_raise(SocketError)
-        expect(ECSCredentials.new(backoff: 0).retries).to be(5)
-      end
+        it 'keeps trying "retries" times, with exponential backoff' do
```

Comment:
Not: can you use backticks here rather than double quotes?

Reply:
I'll just rename the test